packages/block-library/src/list/index.js

```diff
@@ -244,12 +244,12 @@ export const settings = {
 				} );
 			} );
 
-			// this checks for languages that do not typically have square brackets on their keyboards
+			// Check for languages that do not have square brackets on their keyboards.
```

Comment:
These changes are largely unrelated, but I forget why I found this file when working on this patch and just wanted to tidy things up. Sorry 😉